src/Interpreters/Streaming/Aggregator.h

```diff
@@ -650,6 +666,9 @@ class Aggregator final
         size_t window_keys_num;
 
         WindowParamsPtr window_params;
+
+        bool tracking_changes = false;
```

Comment:
Let's comment what's for `tracking_changes` and `tracking_udpates`

Reply:
Probably we can just use one `std::optional<TrackingUpdatesType>` to cover these 2 bools